README.md

```diff
@@ -30,6 +30,12 @@ PGPORT=5432
 # for Postgres versions < 9.4
 PGPASS=password
 ```
+# Email G00dness
+
+EMAIL=hlr.dev.useyouremailN00Bulous@gmail.com
+EMAILSERVICE=Gmail (note the cap)
```

Comment:
I shot the inventor a message about it. I'll let you know what he says. We may be able to simply get a email server and let that car equal it.